### ISSUE TITLE
Support for websiteDisplayGroupName in Catalog API? 

### ISSUE THREAD

**Issue by ShaneOH** [jlevers/selling-partner-api#396]
Hey @jlevers !

Doing a `searchCatalogItems` operation via `GET /catalog/2022-04-01/items` returns the following payload in the `summaries` section (truncated example):

```
         "summaries": [
            {
               "marketplaceId": "ATVPDKIKX0DER",
               ...
               "websiteDisplayGroup": "automotive_display_on_website",
               "websiteDisplayGroupName": "Automotive Parts and Accessories"
            }
         ]
```

The library is translating that to the following:

```
            [summaries] => Array
                (
                    [0] => SellingPartnerApi\Model\CatalogItemsV20220401\ItemSummaryByMarketplace Object
                        (
                            [container:protected] => Array
                                (
                                    [marketplace_id] => ATVPDKIKX0DER
                                    ...
                                    [website_display_group] => automotive_display_on_website
                                )

                        )

                )
```

Namely, `website_display_group_name` is missing from the `ItemSummaryByMarketplace` object. Would it be possible to add this in? 

Thank you!

**Comment by jlevers** [comment]
Hey @ShaneOH, thanks for surfacing this and sorry for the lag time. It's fixed in commit [cb51656](https://github.com/jlevers/selling-partner-api/commit/cb5165698f0d90ae3694b551e13beefb565c9aab), and a new version that includes the fix should be released in the next few days.

**Comment by jlevers** [comment]
This should be fixed in `v5.4.0` -- let me know if you keep encountering this issue.
